### ISSUE TITLE
TypeError Plugin: netlify-cli: memoizeOne is not a function

### ISSUE THREAD

**Issue by trein-homeroom** [netlify/cli#2202]
**Describe the bug**

netlify cli throws error on usage

**To Reproduce**

1. Install latest version via `npm i -g netlify-cli` where latest version is currently v3.21.5
2. Run `netlify dev`
3. See that the CLI exits with an error

**Configuration**

```
$ npx envinfo --system --binaries --npmPackages netlify-cli --npmGlobalPackages netlify-cli
npx: installed 1 in 1.63s

  System:
    OS: Linux 4.19 Ubuntu 18.04.4 LTS (Bionic Beaver)
    CPU: (4) x64 Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz
    Memory: 3.19 GB / 7.29 GB
    Container: Yes
    Shell: 4.4.20 - /bin/bash
  Binaries:
    Node: 10.21.0 - ~/.nvm/versions/node/v10.21.0/bin/node
    Yarn: 1.22.4 - /usr/bin/yarn
    npm: 6.14.6 - ~/.nvm/versions/node/v10.21.0/bin/npm
  npmGlobalPackages:
    netlify-cli: 3.21.5
```

**Expected behavior**

No error

**CLI Output**

```
netlify dev
TypeError Plugin: netlify-cli: memoizeOne is not a function
module: @oclif/config@1.17.0
task: runHook init
plugin: netlify-cli
root: /home/vagrant/.nvm/versions/node/v10.21.0/lib/node_modules/netlify-cli
See more details with DEBUG=*
    TypeError: memoizeOne is not a function
```

**Additional context**

Output above is from my system, but this is breaking on our build in Circleci, so I don't think it's just my system.

**Comment by shukryzablah** [comment]
Our CI run into the same error.

**Comment by unclejustin** [comment]
Came here to post the same error :( downgrading as far as 3.18.0 didn't help

**Comment by unclejustin** [comment]
Wonder if this is related: https://github.com/alexreardon/memoize-one/issues/37

**Comment by trinta** [comment]
Seeing the same issue. The NPM package for memoize-one was updated 2 hours ago, and our issues started at the same time, so that seems to be a likely culprit that the NPM package contains the export change and the function isn't being imported correctly.

**Comment by liam-careerhub** [comment]
Same issue here; tried downgrading as far as 3.1.1 and no improvement. Doing a manual deployment via the Netlify UI for now.

**Comment by ocarcache** [comment]
I have the same issue! This is the output when I run the netlify-cli command:

`Netlify command line tool

VERSION
  netlify-cli/3.21.5 win32-x64 node-v14.6.0

USAGE
  $ netlify [COMMAND]

COMMANDS
  addons      (Beta) Manage Netlify Add-ons
  api         Run any Netlify API method
  build       (Beta) Build on your local machine
  completion  (Beta) Generate shell completion script
  deploy      Create a new deploy from the contents of a folder
  dev         Local dev server
  env         (Beta) Control environment variables for the current site
  functions   Manage netlify functions
  help        display help for netlify
  init        Configure continuous deployment for a new or existing site
  link        Link a local repo or project folder to an existing site on Netlify
  lm          Handle Netlify Large Media operations
  login       Login to your Netlify account
  open        Open settings for the site linked to the current folder
  plugins     list installed plugins
  sites       Handle various site operations
  status      Print status information
  switch      Switch your active Netlify account
  unlink      Unlink a local folder from a Netlify site
  watch       Watch for site deploy to finish

(node:13396) TypeError Plugin: netlify-cli: memoizeOne is not a function
module: @oclif/config@1.17.0
task: runHook init
plugin: netlify-cli
root: C:\Users\tfssvcacct\AppData\Roaming\npm\node_modules\netlify-cli
See more details with DEBUG=*
(Use `node --trace-warnings ...` to show where the warning was created)`

**Comment by alucardu** [comment]
Same issue here.

**Comment by fool** [comment]
Hey folks - while our team looks into this, be aware that we may not get a fix quickly.

However, to unblock you, there are other methods of deployment which will not suffer this problem:

- our legacy CLI - we'll deprecate it someday soon, but for now, should still work:  https://github.com/netlify/netlifyctl
- API.  There are several workflows, one of which is "send a zipfile of your built site":  https://docs.netlify.com/api/get-started/#deploy-via-api
- drag and drop.  This is the worst deployment method since it is dependent on a very solid network connection, your computer having a ton of free memory, and is still prone to failure for larger sites, but if you have a small one, it might work well.  In this case "failure" will look like a stuck deploy, not a partial publish or anything.  More details in [this forums post](https://answers.netlify.com/t/support-guide-my-drag-and-drop-deploy-is-stuck-in-uploading-status/112).

**Comment by keiko713** [comment]
Oops sorry, closing the PR looks like closed this issue! A new (fixed) version is not released yet, but will be shortly! We'll keep you posted.

**Comment by keiko713** [comment]
Updated fool's earlier comment too, but we have released a new version of CLI 3.21.6 which fixes this issue. Please run `npm i -g netlify-cli` to get the latest version of CLI.
Sorry for the inconvenience, and thank you all for reporting this!
Also special thanks to @jsutaria for opening up a PR for the fix, we didn't end up using that PR but contribution was very much appreciated!

**Comment by MarcelGerber** [comment]
We are still running into this issue for our Github Actions-based deploy, even though it is already using version 3.21.6 of `netlify-cli`.

You can see the full run here: https://github.com/owid/covid-19-data/runs/2418123449#step:4:8

```
[4/4] Building fresh packages...
  success Installed "netlify-cli@3.21.6" with binaries:
        - ntl
        - netlify
  Done in 35.01s.

[...]

    TypeError: memoizeOne is not a function
```

**Comment by JGAntunes** [comment]
Hey @MarcelGerber (and others who may still be following this thread). We've released version `3.21.7` and more recently `3.21.8` which should have fixed the issue - https://github.com/alexreardon/memoize-one/issues/116 - permanently both at the `cli` level and at `@netlify/build`. Do let us know if you're still running into some issues though 👍
